pandas/core/indexes/interval.py

```diff
@@ -1140,7 +1140,10 @@ def overlaps(self, other):
 
     @Appender(_index_shared_docs['intersection'])
     @SetopCheck(op_name='intersection')
-    def intersection(self, other, sort=False):
+    def intersection(self,
```

Comment:
typing nitpick, we use single quotes on these annotations, otherwise lgtm. ping on green.

@WillAyd can we systematically check this?